[16.0][ADD]document_url_google_drive

DemchukM opened the pull request:
…ach directly from Google Drive

DemchukM committed:
[IMP]document_url_google_drive: add module to select documents to attach directly from Google Drive

pedrobaeza commented:
Thanks for the contribution. This should be `[ADD]`, not `[IMP]`, as it's adding a new module, not improving an existing one. Ref: https://github.com/OCA/odoo-community.org/blob/master/website/Contribution/CONTRIBUTING.rst#71commit-message

On the other side, please check pre-commit.

DemchukM committed:
[FIX]document_url_google_drive: Moved part of settings from res.users to general settings adn add used and added use module google_account

DemchukM committed:
Merge branch 'OCA:16.0' into 16.0-t3012-document_url_google_drive-add_select_google_drive

DemchukM committed:
[FIX]document_url_google_drive: fix in tests move name parameters

DemchukM committed:
[16.0][ADD]document_url_google_drive: added page Google Api to user profile

DemchukM committed:
[16.0][ADD]document_url_google_drive: added page Google Api to user profile

[FIX]document_url_google_drive: remove tagget decorator in test

[FIX]document_url_google_drive: fix in tests move name parameters

[FIX]document_url_google_drive: fix in tests move name parameters

DemchukM committed:
[FIX]document_url_google_drive: fix in tests move name parameters

DemchukM committed:
[FIX]document_url_google_drive: fix in tests move name parameters

DemchukM committed:
[FIX]document_url_google_drive: remove tagget decorator in test

DemchukM committed:
Merge remote-tracking branch 'origin/16.0-t3012-document_url_google_drive-add_select_google_drive' into 16.0-t3012-document_url_google_drive-add_select_google_drive

# Conflicts:
#	document_url_google_drive/README.rst

ivs-cetmix committed:
Update CONFIGURE.md

DemchukM committed:
Merge remote-tracking branch 'origin/16.0-t3012-document_url_google_drive-add_select_google_drive' into 16.0-t3012-document_url_google_drive-add_select_google_drive

# Conflicts:
#	document_url_google_drive/models/res_users.py

DemchukM committed:
[FIX]document_url_google_drive: fix test

DemchukM committed:
[FIX]document_url_google_drive: fix test

OpenCode commented:
@DemchukM Why there is a deb file? `pandoc-3.1.9-1-amd64.deb`
